tests/tck/features/subgraph/subgraph.IntVid.feature

```diff
@@ -255,7 +255,7 @@ Feature: Integer Vid subgraph
       |                                             |                       | [:like "Marco Belinelli"->"Tony Parker"@0]      |                    |
       |                                             |                       | [:like "Tim Duncan"->"Tony Parker"@0]           |                    |
     Then the result should be, in any order, with relax comparison:
-      | VERTICES         | EDGES     |
+      | _vertices        | _edges    |
```

Comment:
We don't need to do this compatible. According the previous discussion the `yield` clause means fetch the columns of result from previous clause. So `yield edges` means fetch `edges` column of `get subgraph` result.

Reply:
what do you mean don't need todo this compatible